File: app/src/main/java/com/fomchenkovoutlook/artem/android_web_browser_example/WebBrowserViewClient.java
package com.fomchenkovoutlook.artem.android_web_browser_example;

import android.support.annotation.NonNull;
import android.webkit.WebView;
import android.webkit.WebViewClient;
import android.widget.EditText;

// Custom client:
public class WebBrowserViewClient extends WebViewClient {

    private EditText etWebSite;

    public WebBrowserViewClient(@NonNull EditText etWebSite) {
        this.etWebSite = etWebSite;
    }

    @Override
    public void onPageFinished(WebView view, String url) {
        super.onPageFinished(view, url);
        etWebSite.setText(url);
    }
}

File: app/src/main/java/com/fomchenkovoutlook/artem/android_web_browser_example/WebBrowserViewClient.kt
package com.fomchenkovoutlook.artem.android_web_browser_example

import android.support.annotation.NonNull
import android.webkit.WebView
import android.webkit.WebViewClient
import android.widget.EditText

/**
 * Set url in search field
 * @param etWebSite search field
 * */
class WebBrowserViewClient(@NonNull private val etWebSite: EditText): WebViewClient() {

    override fun onPageFinished(view: WebView, url: String) {
        super.onPageFinished(view, url)
        etWebSite.setText(url)
    }

}